# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/base/util-linux/disable_cramfs_tools.diff
# ROCK Linux is Copyright (C) 1998 - 2003 Clifford Wolf
# 
# This program is free software; you can redistribute it and/or modify
# it under the terms of the GNU General Public License as published by
# the Free Software Foundation; either version 2 of the License, or
# (at your option) any later version. A copy of the GNU General Public
# License can be found at Documentation/COPYING.
# 
# Many people helped and are helping developing ROCK Linux. Please
# have a look at http://www.rocklinux.org/ and the Documentation/TEAM
# file for details.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- disk-utils/Makefile.orig	Wed Jan 15 20:15:21 2003
+++ disk-utils/Makefile	Wed Jan 15 20:15:49 2003
@@ -20,7 +20,7 @@
 
 ETC=	fdprm
 
-MAYBE=  setfdprm raw fsck.cramfs mkfs.cramfs
+MAYBE=  setfdprm raw
 
 ifneq "$(HAVE_FDUTILS)" "yes"
 USRBIN:=$(USRBIN) setfdprm
@@ -32,21 +32,7 @@
 MAN8:=$(MAN8) raw.8
 endif
 
-ifeq "$(HAVE_ZLIB)" "yes"
-SBIN:=$(SBIN) fsck.cramfs mkfs.cramfs
-endif
-
 all: $(SBIN) $(USRBIN)
-
-fsck.cramfs: fsck.cramfs.o
-	$(CC) $(LDFLAGS) -o fsck.cramfs fsck.cramfs.o -lz
-
-mkfs.cramfs: mkfs.cramfs.o $(LIB)/md5.o
-	$(CC) $(LDFLAGS) -o mkfs.cramfs mkfs.cramfs.o $(LIB)/md5.o -lz
-
-mkfs.cramfs.o: $(LIB)/md5.h
-
-fsck.cramfs.o mkfs.cramfs.o: cramfs.h
 
 fsck.minix.o mkfs.minix.o: bitops.h minix.h
 
